lms/templates/dashboard.html

```diff
@@ -103,12 +103,13 @@ <h2 class="header-courses">${_("My Courses")}</h2>
       </ul>
     % else:
       <section class="empty-dashboard-message">
-        <p>${_("Looks like you haven't enrolled in any courses yet.")}</p>
+        <p>${_("It looks like you aren't yet enrolled in any courses!")}</p>
 
         % if settings.FEATURES.get('COURSES_ARE_BROWSABLE'):
-          <a href="${marketing_link('COURSES')}">
-            ${_("Find courses now!")}
+          <a class="action action-reverify"href="${marketing_link('COURSES')}">
+            ${_("Explore edX Courses!")}
```

Comment:
If we use an exclamation point at the end of the above message, I wouldn't include one here.